### PR TITLE
feat:add image and Input

### PR DESCRIPTION
**Pull request by BISINIGISHIVANI**
image
image
image

Add input and Image responsive

### CODE REVIEW

File: components/Input/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <input type="text" name="last-name" id="last-name"placeholder="enter your  last name">
+        </form>
+        <form action=""class="form-flex validatin-form-size">
+            <h4>Input validation and Error</h4>
+            <label for="gmail">Gmail*<input type="text" name="gmail" id="gmail"placeholder="enter your gmail"required></label>
+            <label id="label-pwd"for="phone-number">Phone Number<input type="text" name="phone-number" id="phone-number"placeholder="enter your phone-number"></label>
+            <input type="button" name="submit" id="submit"value="submit">
+            <div id="output"></div>
+        </form>
+        <script src="./script.js"></script>
```
Comment (shaik-saad) [inline components/Input/index.html]: you can move the script tag out of the section tag, script tags should be place at the bottom of the body tag.

**Review by shaik-saad — APPROVED** [review]
Good work, you can go ahead with the merge.

**Comment by shaik-saad** [comment]
For screenshots you can use windows snipping tool, shortcut to open the tool: "windows key + shift + s".
Also you can screenshot the whole screen with: "windows key + print screen (prt sc) key"

File: components/Input/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <form action="#"class="form-flex form-size">
+        <h4>Input Textbox</h4>
+        <label for="fisrt-name">First Name:</label>
+        <input type="text" name="fisrt-name" id="fisrt-name"placeholder="enter your first name">
+        <label for="last-name">Last Name:</label>
+        <input type="text" name="last-name" id="last-name"placeholder="enter your  last name">
+        </form>
+        <form action=""class="form-flex validatin-form-size">
+            <h4>Input validation and Error</h4>
+            <label for="gmail">Gmail*<input type="text" name="gmail" id="gmail"placeholder="enter your gmail"required></label>
```
Comment (Husain01) [inline components/Input/index.html]: change type to type="email" as html has inbuilt validation for emails.

**Comment by Husain01** [comment]
You could style the inputs a bit other than that everything looks good.

**Review by Husain01 — APPROVED** [review]

File: components/Input/index.html (hunk 2 above)
Comment (BISINIGISHIVANI) [reply components/Input/index.html]: okay i update it.

File: components/Input/index.html (hunk 1 above)
Comment (BISINIGISHIVANI) [reply components/Input/index.html]: i  will change it

**Pull request merged**